docs-search/client-halogen/spago.yaml

```diff
@@ -38,7 +38,6 @@ package:
   bundle:
     type: "app"
     minify: true
-    sourceMaps: true
```

Comment:
And immediately we have a winner!

Turns out `spago bundle` has a command-line parameter for source maps, but not a config parameter. Another bug.

Reply:
Or not a bug? Technically this can be specified in extra args, as I did here, but this is asymmetry. Not sure what the overarching philosophy is here.

Reply:
Great catch!

Overarching philosophy is that we try to keep flags and config params as close to each other as possible. There are some notable exceptions, but this one is a bug: we should have a config parameter as well.